packages/astro/test/units/i18n/astro_i18n.test.js

```diff
@@ -636,6 +636,7 @@ describe('getLocaleAbsoluteUrl', () => {
 				...config.experimental.i18n,
 				trailingSlash: 'never',
 				format: 'directory',
+				site: 'https://example.com',
```

Comment:
This file was never run as it didn't have the `.test.js` postfix. I added it but it failed here, which I fixed it by passing `site` here.